crates/kitsune_p2p/direct/src/v1.rs

```diff
@@ -899,55 +897,56 @@ async fn handle_query_op_hashes(
     //        we'll want an api to just get the hashes
     let mut entries = vec![];
 
-    for (agent, arcset) in agents {
-        let agent = KdHash::from_kitsune_agent(&agent);
-        let es = kdirect
-            .persist
-            .query_entries(root.clone(), agent, window.clone(), arcset)
-            .await?;
-        entries.extend(es.into_iter());
-    }
-
-    let mut entries: Vec<_> = entries
-        .into_iter()
-        .map(|e| e.hash().clone().to_kitsune_op_hash())
-        .collect();
-    entries.sort();
-    entries.dedup();
+    // let agents = todo!();
```

Comment:
is this intentionally left commented out?

Reply:
Fixed in https://github.com/holochain/holochain/commit/140576f89e08d81a09e9c21a2a84d9d17a66482b